examples/hotrod/README.md

```diff
@@ -65,6 +65,17 @@ docker run \
 
 Then open http://127.0.0.1:8080
 
+### Run HotROD/Jaeger all-in-one from kubernetes
```

Comment:
I would move this right after the section `### Run everything via docker-compose`

Reply:
Done in https://github.com/jaegertracing/jaeger/pull/3645/commits/05330372471e57e49436af21d738741bd77674f1